client/src/App.js

```diff
@@ -21,7 +21,11 @@ function App() {
 
         <Route exact path='/my-lists'>
           <Lists />
-          
+                  
+        </Route>
```

Comment:
hmm, not sure if we need this route anymore...customer is not happy with is..please remove and update.